src/NServiceBus.AcceptanceTests/Mutators/Issue_1980.cs

```diff
@@ -8,18 +8,17 @@
 
     public class Issue_1980 : NServiceBusAcceptanceTest
     {
+        static Context testContext = new Context();
```

Comment:
This is no longer possible with the changes done on the async work for the ATT framework. Why did you change that?